app/Http/Controllers/CompanyController.php

```diff
@@ -20,4 +21,10 @@ public function overview()
     {
         return view('company.overview');
     }
+
+    public function destroy(Company $company)
+    {
+        $company->delete();
+        return back();
+    }
 }
```

Comment:
Maybe do a permission check here just to be certain. I know you might now actually display the destroy button for people that don't have the permission but double layers of security never harmed anybody.

Reply:
Truetrue